ebu_tt_live/xsd1.1/ebutt_datatypes.xsd

```diff
@@ -44,12 +44,17 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
 		</xs:restriction>
 	</xs:simpleType>
 	<xs:simpleType name="lineHeightType">
-		<xs:union memberTypes="ebuttdt:lengthType">
+		<xs:union>
 			<xs:simpleType>
 				<xs:restriction base="xs:token">
 					<xs:enumeration value="normal"/>
 				</xs:restriction>
 			</xs:simpleType>
+		<xs:simpleType>
+			<xs:restriction base="xs:string">
+				<xs:pattern value="\d*\.?\d+(px|c|%)"/>
+			</xs:restriction>
```

Comment:
My temporary solution is to not use length type and simply define the right regular expression. It is not the best solution, but adding `nonNegativeLengthType` now in the XSD, while it is not already in the spec would not be a good idea I think